### ISSUE TITLE
Add Table of Contents to OUSD Blog

### ISSUE THREAD

**Issue by rmcn15** [OriginProtocol/ousd.com#105]
A table of contents on the OUSD blog will help us hit Keywords again within H2s/H3s.

Here's an example on the staging site: https://staging.ousd.com/litepaper

If this can automatically pick up H2s for headers and H3s as subheaders within the ToC that would be perfect. Alternatively I can manually add these sections in the CMS.

**Comment by franckc** [comment]
CC @micahalcorn for prioritization

CC @HrikB for commenting on the eng scope

**Comment by HrikB** [comment]
Yeah should definitely be possible. It currently pulls each section from the cms... you can see how it is configured in the OUSD - Litepaper collection type. But theoretically it is possible to just read the h2s and h3s of the page and construct that data by itself. 

The thing is I will need to attach a reference to each h2/h3 so the ToC navigates properly which I should be able to do from parsing the string... but if that doesn't work we may need to separate each section.
